.github/workflows/ubuntu.yml

```diff
@@ -88,12 +110,23 @@ jobs:
           [ -d "${LOCAL_BUILDS}/rnp-build/src/tests" ]
           [ -d "${LOCAL_BUILDS}/rnp-build/src/tests/googletest-build" ]
           [ ! -d "${LOCAL_BUILDS}/rnp-build/src/tests/googletest-src" ]
+          
   cmake-offline-googletest:
+    strategy:
+      matrix:
+        cc: ["gcc"]
```

Comment:
I don't think that we should add clang test here - this test is just to check whether offline googletest cmake define works well.

Reply:
Frankly speaking, I have no idea what is really required. In 99%+ cases everything that works with gcc will also work with clang. 
For other workflows I was debugging and fixing ci shell scripts which are .... well, a little bit difficult to maintain.
So if there is a list of tests that have to be extended to clang I would rather  delete extra staff before any reviews and save some time for everybody.

Thank you!

Reply:
Considering this specific test, we check that offline googletest cmake define works well when CC/CXX is set to clang/clang++ which activates different toolchain.  As I mentioned above it really tests ci scripts - run.sh and others.

Reply:
These tests are commented